targets/TARGET_Cypress/TARGET_PSOC6/TESTS/host_tests/softap_basic.py

```diff
@@ -0,0 +1,231 @@
+"""
+mbed SDK
+Copyright (c) 2011-2019 ARM Limited
+
```

Comment:
please add SDPX identifier to new files

when updating, fix the year to 2020 and remove mbed SDK line as well

Reply:
Done.